### PR TITLE
Add continuous integration

### PR DESCRIPTION
**Pull request by heewa**
This will allow you :smiley:  to hook :fishing_pole_and_fish: up :arrow_up: [Travis CI](https://travis-ci.org), which will run tests :white_check_mark: on your project :memo: , to let you know if there are any problems :rage:  with your build :wrench: 

image

:eyes: @dvdokkum @andrewwitherspoon @tomger @dbow

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.7"
```
Comment (dbow) [inline .travis.yml]: I think an older version of python would be more subparr

**Comment by dbow** [comment]
LGTM

**Comment by heewa** [comment]
@dbow @dvdokkum I addressed all comments in the review. Please merge at your earliest convenience, as per to your approval. :trophy: :star2:

**Comment by danielmcgrath** [comment]
:trophy:

**Pull request merged**